scripts/doc/create_doc_task_generator.py

```diff
@@ -521,17 +521,32 @@ def main(argv=sys.argv[1:]):
             'rsync',
             # the following are required by rosdoc_lite
             'doxygen',
-            'python-catkin-pkg-modules',
-            'python-epydoc',
-            'python-kitchen',
-            'python-rospkg',
-            'python-sphinx',
-            'python-yaml',
             # since catkin is not a run dependency but provides the setup files
             get_os_package_name(args.rosdistro_name, 'catkin'),
             # rosdoc_lite does not work without genmsg being importable
             get_os_package_name(args.rosdistro_name, 'genmsg'),
         ]
+
+        if '3' == condition_context['ROS_PYTHON_VERSION']:
+            # the following are required by rosdoc_lite
+            debian_pkg_names.extend([
+                'python3-catkin-pkg-modules',
+                'python3-kitchen',
+                'python3-rospkg-modules',
+                'python3-sphinx',
+                'python3-yaml'])
+        else:
+            print('Unknown python version', condition_context)
```

Comment:
That message isn't necessarily true. The Python version might be known - just not `3`.

Reply:
Only print if python version is not `2` in 949059331f282cd2275002be303c68f342e277fd